client/src/main/scala/drt/client/components/FlightsWithSplitsTable.scala

```diff
@@ -28,7 +28,8 @@ object FlightsWithSplitsTable {
                    arrivalSources: Option[(UniqueArrival, Pot[List[Option[FeedSourceArrival]]])],
                    hasArrivalSourcesAccess: Boolean,
                    viewMode: ViewMode,
-                   pcpPaxFn: Arrival => Int
+                   pcpPaxFn: Arrival => Int,
+                   portCode: PortCode
```

Comment:
we should probably use this "hasTransfer" flag https://github.com/UKHomeOffice/drt-v2/blob/master/shared/src/main/scala/drt/shared/HasAirportConfig.scala#L253

Rather than hard code LHR in here.